modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -126,7 +126,7 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers with Transa
         Transfer,
         none,
         AccountIdentifier(address, none).some,
-        Amount(AmountValue(-1L), Currency(CurrencySymbol("DAG"), CurrencyDecimal(8L))).some
+        Amount(AmountValue(-1L), currency.Currency(currency.CurrencySymbol("DAG"), currency.CurrencyDecimal(8L))).some
```

Comment:
bad auto import

Reply:
This change was because of the following linter error.

```
/Users/chuong/Documents/dev/constellationnetwork/tessellation/modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala:164:43: reference to CurrencySymbol is ambiguous;
[error] it is imported twice in the same scope by
[error] import org.tessellation.rosetta.domain.currency._
[error] and import org.tessellation.rosetta.domain._
[error]         Amount(AmountValue(-1L), Currency(CurrencySymbol("DAG"), CurrencyDecimal(8L))).some
```

Reply:
Yup, so don't import it twice and you should be good

Reply:
Without importing `org.tessellation.rosetta.domain.currency._`, `Currency`, `CurrencySymbol`, etc. don't resolve. If I take out `org.tessellation.rosetta.domain._`, I lose `RosettaError`, `TransactionIdentifier`, etc. Using `currency.` seems like the least bad way to solve the problem.

Reply:
You can import the exact classes you need

Reply:
If I import all the `currency` classes I need, I end up with

`import org.tessellation.rosetta.domain.currency.{Currency, CurrencySymbol, CurrencyDecimal, DAG}`

after the linter is run, that becomes

`import org.tessellation.rosetta.domain.currency._`

and then the `ambiguous reference` errors pop up.